src/scss/02_atoms/_accordion.scss

```diff
@@ -117,6 +120,11 @@ $darkAccordionConfig: (
   width: 100%;
 }
 
+
+.accordion__toggle__content {
+  padding-right: get($accordionConfig, accordionToggle, content, paddingRight);
```

Comment:
Questo sarebbe da rimuovere perchè rappresenta un breaking change su tutti gli accordion che esistono.
Lo farei a parte facendo una PR per fixare tutti gli implementatori: 
- Jobs (da verificare)
- Prima 
- Pyxis site 
- Frontale 
- Hutch 
- Crash (da verificare)

Reply:
In realtà però va solo ad aggiungere una caratteristica e se non utilizzi questa classe o definisci il blocchetto di HTML tutto dovrebbe funzionare uguale o sbaglio?

Reply:
Vero, però significa che in tutte le implementazioni avremo il modale con la vecchia struttura, e questo non dovrebbe accadere